setup.py

```diff
@@ -78,6 +78,9 @@
     long_description = pypandoc.convert_file('README.md', 'rst')
 except ImportError:
     pass
+except RuntimeError as e:
```

Comment:
Change this to handle all exceptions ("Exception as e") and simply print it out and continue.